app/src/test/java/mozilla/lockbox/presenter/ItemDetailPresenterTest.kt

```diff
@@ -18,15 +20,38 @@ import org.junit.Assert.fail
 import org.junit.Before
 import org.junit.Test
 import org.mozilla.sync15.logins.ServerPassword
+import org.mockito.Mockito
 
 class ItemDetailPresenterTest {
     class FakeView : ItemDetailView {
+
         override var itemId: String? = null
         var item: ItemDetailViewModel? = null
+        val tapStub: PublishSubject<Unit> = PublishSubject.create<Unit>()
+
+        override val btnUsernameCopyClicks: Observable<Unit>
+            get() = tapStub
+
+        override val btnPasswordCopyClicks: Observable<Unit>
+            get() = tapStub
+
+        override val btnTogglePasswordClicks: Observable<Unit>
+            get() = tapStub
+
+        override val editUsername: EditText
+            get() = TODO("not implemented") // To change initializer of created properties use File | Settings | File Templates.
+        override val editPassword: EditText
+            get() = TODO("not implemented") // To change initializer of created properties use File | Settings | File Templates.
 
         override fun updateItem(item: ItemDetailViewModel) {
             this.item = item
         }
+
+        override fun copyNotification(strId: Int) {
```

Comment:
Nit: please comment empty blocks. 

(otherwise, this looks like you've forgotten to `git add` something.